pandas/io/parquet.py

```diff
@@ -571,6 +575,12 @@ def read_parquet(
 
         .. versionadded:: 2.1.0
 
+    to_pandas_kwargs : dict | None, default None
+        Keyword arguments to pass through to ``pyarrow.Table.to_pandas``
```

Comment:
```suggestion
        Keyword arguments to pass through to :func:`pyarrow.Table.to_pandas`
```

We can actually use the above syntax to have it link to the pyarrow docs automatically (so people can more easily check which keywords are available)